### PR TITLE
Fixed dependencies

### PR DESCRIPTION
**Pull request by christianascone**
I fixed missing and broken dependencies.
Without them I was not able to run a fresh install.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# HLTB Mobile App
+
+![hltb Version][hltb-image]
+
+This is a open-source, unofficial, and non-commercial mobile app for [How Long To Beat](https://howlongtobeat.com).
+
+![](src/assets/img/hltb_icon.png)
+
+
```
Comment (nmoya) [inline README.md]: Since you recently cloned the project, can you please list the steps to build the project?

File: app.json [hunk 2]
```diff
@@ -1,4 +1,4 @@
 {
-  "name": "How Long To Beat",
-  "displayName": "How Long To Beat"
-}
+  "name": "hltb",
+  "displayName": "hltb"
```
Comment (nmoya) [inline app.json]: Why did we had to change this?

File: README.md (hunk 1 above)
Comment (christianascone) [reply README.md]: Hi @nmoya. Yeah, I'm going to clone a fresh copy and re-build it in order to update README with every necessary step.

Comment (nmoya) [reply README.md]: Thank you so much!

File: app.json (hunk 2 above)
Comment (christianascone) [reply app.json]: I think react changed this when I ran the project.
I will revert it because I suppose it does not need to be changed.

**Comment by christianascone** [comment]
I fixed app name and displayName.
I also updated readme file with requirements and installation instructions (now it should runnable in a few steps).

**Pull request merged**